src/Test/ItemsTestCase.php

```diff
@@ -40,6 +40,24 @@
         $this->assertInstanceOf(Items::class, $this->items);
     }
 
+    /**
+     * @test
+     */
+    final public function it_is_traversable() : void
+    {
+        for ($i = 1; $i <= 500; $i++) {
+            $this->items->add($this->generateItemVersion("item{$i}", 1));
+            $this->items->add($this->generateItemVersion("item{$i}", 2));
+        }
+
+        $this->assertCount(500, $this->items);
+        $i = 0;
+        foreach ($this->items as $item) {
```

Comment:
so the iterator is ascending as a default ordering, is it consistent with the other access patterns like `list()`?